### PR TITLE
Rhino default linters

### PR DESCRIPTION
**Pull request by radbasa**
Issue #4

Branched from `migrate-existing-linters` PR #7 

## Description
* Add `rhino_default_linters`

`rhino dot.lintr`
```yaml
linters:
  linters_with_defaults(defaults = box.linters::rhino_default_linters)
```

## Definition of Done
- [ ] The change is thoroughly documented.
- [ ] The CI passes (`R CMD check`, linter, unit tests, spelling).
- [ ] Any generated files have been updated (e.g. `.Rd` files with `roxygen2::roxygenise()`)

### CODE REVIEW

**Comment by radbasa** [comment]
```
linters:
  linters_with_defaults(
    defaults = box.linters::rhino_default_linters,
    line_length_linter = lintr::line_length_linter(140)
)
```

**Comment by radbasa** [comment]
```YAML
linters:
  linters_with_rhino_defaults(
    line_length_linter = lintr::line_length_linter(140)
)
```

```
box.linters::linters_with_rhino_defaults <- function(...) {
  lintr::linters_with_defaults(..., defaults = box.linters::rhino_defaults)
}
```

File: R/defaults.R [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#' @examples
+#' linters <- lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters)
+#'
+#' names(linters)
+#'
+#' @export
+# nolint end
+rhino_default_linters <- lintr::modify_defaults(
+  defaults = lintr::default_linters,
+  line_length_linter = lintr::line_length_linter(100),
```
Comment (radbasa) [inline R/defaults.R:16]: Remove. Leave in `rhino...lintr`

Comment (radbasa) [reply R/defaults.R:16]: Implementing this in PR #35 . `defaults.R` file changed name to `zzz.R`. Don't want to deal with upstream merge issues.

**Review by jakubnowicki — APPROVED** [review]
LGTM

**Pull request merged**